### PR TITLE
Improve trino-product-tests docs

### PR DESCRIPTION
**Pull request by brandboat**
- Replace Presto with Trino
- Fix incorrect cmd in product-tests readme

### CODE REVIEW

File: testing/trino-product-tests/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ broken.
 * Install [Docker for Mac](https://docs.docker.com/docker-for-mac/)
 
 * Add entries in `/etc/hosts` for all services running in docker containers:
-`hadoop-master`, `postgres`, `cassandra`, `presto-master`.
+`hadoop-master`, `postgres`, `cassandra`, `trino-master`.
```
Comment (kokosing) [inline testing/trino-product-tests/README.md:37]: @wendigo do we still use `trino-master` or now `coordinator`?

Comment (findepi) [reply testing/trino-product-tests/README.md:37]: `presto-master` this still is -- https://github.com/trinodb/trino/blob/cea58426e0a8f4cfe330bc805f88d12ab2d3fc4e/testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java#L18-L19

Comment (wendigo) [reply testing/trino-product-tests/README.md:37]: Still presto-master

**Comment by kokosing** [comment]
Merged, thanks!

File: testing/trino-product-tests/README.md (hunk 1 above)
Comment (kokosing) [reply testing/trino-product-tests/README.md:37]: Right, this is going to be updated with: https://github.com/trinodb/docker-images/pull/89.

I merged this too quickly. ; /

Comment (kokosing) [reply testing/trino-product-tests/README.md:37]: https://github.com/trinodb/trino/pull/7008

**Comment by brandboat** [comment]
Thanks for the quick review !

**Pull request merged**